vmdb/db/migrate/20150507102612_create_container_node_conditions.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerNodeConditions < ActiveRecord::Migration
+  def up
+    create_table :container_node_conditions do |t|
+      t.belongs_to :container_node, :type => :bigint
```

Comment:
@dkorn / @simon3z  - no ems_ref in this entity? afaik we did ems_ref for entities such as container definition/container, iirc it was advised by @Fryguy at the time. maybe it's no longer the recommended practice?